From 9c3329bf22e42dd821ddba0b2cbe755996f82c14 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Sandro=20Knau=C3=9F?= <hefee@debian.org>
Date: Sun, 17 Feb 2019 10:50:15 +0100
Subject: [PATCH] On Unix systems shared libraries have a abi version.

The ABI Version is the Interface version of that library on a users
system the ABI Version less so file is not installed. Also we normally
need to recompile the code against a new ABI Version of libsecret. This
fix make it possible to use qtkeychain in a Ubunut/Debian system.
---
 libsecret.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/libsecret.cpp b/libsecret.cpp
index b76f4c5..aed91a0 100644
--- a/libsecret.cpp
+++ b/libsecret.cpp
@@ -305,7 +305,7 @@ bool LibSecretKeyring::deletePassword(const QString &key, const QString &service
 }
 
 LibSecretKeyring::LibSecretKeyring()
-    : QLibrary("secret-1")
+    : QLibrary(QStringLiteral("secret-1"), 0)
 {
 #ifdef HAVE_LIBSECRET
     if (load()) {
-- 
2.27.0

